corehq/apps/cloudcare/static/cloudcare/js/markdown.js

```diff
@@ -140,7 +140,18 @@ hqDefine('cloudcare/js/markdown', [
             // lazy init to avoid dependency order issues
             md = initMd();
         }
-        return md.render(DOMPurify.sanitize(text || "").replaceAll("&#10;", "\n"));
+        var rendered = md.render(DOMPurify.sanitize(text || "").replaceAll("&#10;", "\n"))
+        rendered = appendExtraStyleClass(rendered, '<p>', 'mb-0')
+        rendered = appendExtraStyleClass(rendered, '<h6>', 'mb-0')
```

Comment:
What makes h6 special? Is this not a problem with the other head line levels?

Reply:
<h6> is what's used for the text under the column header (the (update x days ago) caption) and it's specifically so that that caption doesn't create too much space between the headers and the sub case tile rows below. As far as I'm aware there aren't any other head line levels used in that space.

Reply:
I am just wondering about markdown configured by the users.

Reply:
Made it more restrictive to the particular scenario used by delivery by putting these modifications behind a conditional that checks specifically for the header style they use (<p><strong>) so hopefully won't be universally applied whenever another user decides to use <h6>